src/OpenTelemetry.Exporter.ZPages/ZPagesExporterStatsHttpServer.cs

```diff
@@ -108,17 +110,69 @@ private void WorkerThread()
             {
                 while (!this.tokenSource.IsCancellationRequested)
                 {
-                    var ctxTask = this.httpListener.GetContextAsync();
+                    Task<HttpListenerContext> ctxTask = this.httpListener.GetContextAsync();
                     ctxTask.Wait(this.tokenSource.Token);
 
                     var ctx = ctxTask.Result;
 
                     ctx.Response.StatusCode = 200;
                     ctx.Response.ContentType = ZPagesStatsBuilder.ContentType;
 
-                    using var output = ctx.Response.OutputStream;
-                    using var writer = new StreamWriter(output);
-                    writer.WriteLine("Span Count : " + this.spanProcessor.GetSpanCount());
+                    using (Stream output = ctx.Response.OutputStream)
+                    {
+                        using (var writer = new StreamWriter(output))
+                        {
+                            writer.WriteLine("<!DOCTYPE html>");
+                            writer.WriteLine("<html><head><title>RPC Stats</title>" +
+                                             "<meta charset=\"utf-8\">" +
+                                             "<link rel=\"stylesheet\" href=\"https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css\">" +
+                                             "<script src=\"https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js\"></script>" +
+                                             "<script src=\"https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js\"></script></head>");
+                            writer.WriteLine("<body><div class=\"col-sm-1\"></div><div class=\"container col-sm-10\"><div class=\"jumbotron table-responsive\"><h1>RPC Stats</h2>" +
+                                             "<table class=\"table table-bordered table-hover table-striped\">" +
+                                             "<thead style=\"color: white;background-color: Teal;\"><tr><th>Span Name</th><th>Total Count</th><th>Count in last minute</th><th>Count in last hour</th><th>Average Latency (ms)</th>" +
+                                             "<th>Average Latency in last minute (ms)</th><th>Average Latency in last hour (ms)</th><th>Total Errors</th><th>Errors in last minute</th><th>Errors in last minute</th><th>Last Updated</th></tr></thead>" +
+                                             "<tbody style=\"background-color: white;\">");
```

Comment:
I have removed the explicit styling for now. Adding a css file in the code did not work since the HTML looks for a css which is hosted on the server, instead of finding it in the project.
![image](https://user-images.githubusercontent.com/13450864/82726736-e8a14080-9c9a-11ea-91b2-9a0920550f96.png)